packages/compiler/src/core/compiler-code-fixes/triple-quote-indent.codefix.ts

```diff
@@ -0,0 +1,69 @@
+import { isWhiteSpaceSingleLine } from "../charcode.js";
+import { defineCodeFix, getSourceLocation } from "../diagnostics.js";
+import { splitLines } from "../helpers/syntax-utils.js";
+import type { DiagnosticTarget } from "../types.js";
+
+export function createTripleQuoteIndentCodeFix(diagnosticTarget: DiagnosticTarget) {
```

Comment:
the point of changing where you called was  this could also be changed to `SourceLocation` and remove the call to `getSourceLocation` inside.